nucliadb/nucliadb/writer/api/v1/field.py

```diff
@@ -108,19 +107,24 @@ async def finish_field_put(
     writer: BrokerMessage,
     toprocess: PushPayload,
     partition: int,
-    wait_on_commit: bool,
 ) -> Optional[int]:
     # Create processing message
     transaction = get_transaction_utility()
     processing = get_processing()
-
-    processing_info = await processing.send_to_process(toprocess, partition)
-
-    writer.source = BrokerMessage.MessageSource.WRITER
-    set_processing_info(writer, processing_info)
-    await transaction.commit(writer, partition, wait=wait_on_commit)
-
-    return processing_info.seqid
+    try:
+        writer.source = BrokerMessage.MessageSource.WRITER
+        await transaction.commit(writer, partition, wait=True)
+    except TransactionCommitTimeoutError:
+        raise HTTPException(
+            status_code=501, detail="Inconsistent write. Commit timeout"
```

Comment:
What about something more descriptive?

```suggestion
            status_code=501, detail="Inconsistent write. This resource will never be processed and may be never stored"
```